validateSAID.py

```diff
@@ -1,18 +1,19 @@
 def check(id):
+    not_valid = -1
     id = id.strip()
     if (id.isdigit() is False):
-        return False
+        return not_valid
```

Comment:
The PR is ok, but can you please remove `not_valid` and replace it with -1, no need for this variable

Reply:
Readme file says that -1 for incorrect NID so.

Instead of using **both states** (-1 and False ), it's good to use **any of one**.

I also looked in other languages file that return -1 instead of false so I make it -1

Below is reference screen cap of readme.

![image](https://user-images.githubusercontent.com/43564590/97436017-5f212900-1932-11eb-9a6c-2544040da15a.png)

Reply:
Agree, but can we remove the variable `not_valid` and return `-1` directly, no need for this variable

Reply:
Done.